### PR TITLE
Add recipe for pyerfa

### PR DESCRIPTION
**Pull request by mwcraig**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mwcraig** [comment]
@astrofrog-conda-forge -- can you please confirm you are willing to be listed as a maintainer here?

File: recipes/pyerfa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - setuptools_scm
+    - jinja2 >=2.10.3
+    - python
+    - numpy 1.16.*
```
Comment (scopatz) [inline recipes/pyerfa/meta.yaml:25]: Does this need to be pinned so tightly. Wouldn't >=1.16 work?

Comment (mwcraig) [reply recipes/pyerfa/meta.yaml:25]: I was honestly a little confused. The minimum supported numpy version is 1.16, so I thought I should build against that one, but the runtime constraint is `>=1.16` and am happy to change it in host too.

**Comment by astrofrog-conda-forge** [comment]
I'm happy to be listed as a maintainer!

File: recipes/pyerfa/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/pyerfa/meta.yaml:25]: I think this is fine, just curios

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**